### PR TITLE
Docker Cloud: update tag mapping label, some typo fixes

### PR DESCRIPTION
**Pull request by sanscontext**
Updates the Tag Mapping label to the new "Build Rules". Plus some copy edits and typo fixes.

@pkennedyr @pchico83 

Signed-off-by: LRubin <lrubin@docker.com>

### CODE REVIEW

File: docker-cloud/builds/automated-build.md [hunk 1]
```diff
@@ -90,13 +84,14 @@ repository service where the image's source code is stored.
...
 
-    * Specify the **Dockerfile location** as a path relative to the root of the source code repository. (If the Dockerfile is at the repository root, leave this path set to `/`.)
-
     * Enter the tag to apply to Docker images built from this source.
+
         If you configured a regex to select the source, you can reference the
         capture groups and use its result as part of the tag. To learn more, see
         [regexes](automated-build.md#regexes-and-automated-builds).
 
+    * Specify the **Dockerfile location** as a path relative to the root of the source code repository. (If the Dockerfile is at the repository root, leave this path set to `/`.)
```
Comment (pchico83) [inline docker-cloud/builds/automated-build.md]: it is relative to the build context, not to the root

Comment (sanscontext) [inline docker-cloud/builds/automated-build.md]: Context isn't live yet. That update will happen later.

Comment (pchico83) [inline docker-cloud/builds/automated-build.md]: ok, as long as it is changed before going live.

Comment (sanscontext) [inline docker-cloud/builds/automated-build.md]: That'll be in a separate PR, as discussed. :)
